.buildkite/steps/test-go-fmt.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+if [[ $(gofmt -l ./ | head -c 1 | wc -c) != 0 ]]; then
+  echo "The following files haven't been formatted with \`go fmt\`:"
+  gofmt -l ./
+  echo
+  echo "Fix this by running \`go fmt ./...\` locally, and committing the result."
+  exit 1
+fi
```

Comment:
there's one of these lovely hand-rolled things found in every go pipeline 😂

Reply:
Time for a gofmt Buildkite plugin? :)